adding ps command

mikebrow opened the pull request:
Here's the start of some code to add a simple ps command to runc to address issue #380 

Thought I'd put this up and ask for comments.

Signed-off-by: Mike Brown <brownwm@us.ibm.com>

hqhq commented on ps.go:
```diff
@@ -0,0 +1,92 @@
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
```
We can use `--root` to change the state directory for different containers, I think we may need some constant directory for information that `runc ps` needs.

mikebrow replied:
Or let users that use a different root directory enter the --root option when they run the runc command?

mikebrow commented:
I could modify the usage instructions to say "lists containers started by runc with the given root"...

mikebrow commented:
Added a created column to the ps command, similar to the docker ps command created column which indicates how long the container has been running.  Also modified the usage instructions to refer to the fact that the ps is for a given root.

mikebrow commented:
@duglin suggests showing the time stamp for when the process was started vs the time running information.

crosbymichael commented:
We should name this `list` and not `ps`

mikebrow commented:
I'm re-basing this PR right now.  Will change the command to list or wdyt of ls?

crosbymichael commented:
In the future you can just change your original pr by force pushing to your branch instead of making new prs ;)

mikebrow commented:
:-) Ok.
